test/elm/datetime/test.coffee

```diff
@@ -154,33 +156,78 @@ describe 'Time', ->
 describe 'Today', ->
   @beforeEach ->
     setup @, data
+    @ctx = new PatientContext(@ctx.library, @ctx.patient, @ctx.codeService, @ctx.parameters)
```

Comment:
Wouldn't it be better to pass in an actual hard-coded execution date time (e.g. `2012-04-15T03:22:56.123-04:00` and then below test against the expected constants (e.g., `2012` rather than `@ctx.getExecutionDateTime().year`)?

Reply:
Also same for other similar tests you changed?